### PR TITLE
add in logstash hr

### PR DESCRIPTION
**Pull request by cakeben**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: k8s/demo/common/ccd/ccd-logstash.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      logstash.yml: |
+        xpack.monitoring:
+          enabled: true
+        xpack.monitoring.elasticsearch:
+          url: ["http://ccd-data-0.service.core-compute-demo.internal:9200","http://ccd-data-1.service.core-compute-demo.internal:9200","http://ccd-data-2.service.core-compute-demo.internal:9200"]
+    logstashPipeline:
+      logstash.conf: |
+        input {
+          jdbc {
+            jdbc_connection_string => "${DB_URL}"
```
Comment (mario-paniccia) [inline k8s/demo/common/ccd/ccd-logstash.yaml:32]: when are these placeholders replaced?

**Review by mario-paniccia — DISMISSED** [review]
👍

File: k8s/demo/common/ccd/ccd-logstash.yaml (hunk 1 above)
Comment (cakeben) [reply k8s/demo/common/ccd/ccd-logstash.yaml:32]: I just need to set them as env vars, but need to grab the env vars from vault... will all be done shortly enough

File: k8s/demo/common/ccd/ccd-logstash.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    flux.weave.works/automated: "true"
+spec:
+  releaseName: ccd-logstash
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    git: git@github.com:elastic/helm-charts
+    ref: master
+    path: tree/master/logstash
```
Comment (adusumillipraveen) [inline k8s/demo/common/ccd/ccd-logstash.yaml:17]:
```suggestion
    repository: https://helm.elastic.co
    name: elasticsearch
    version: 6.8.13
```

**Review by adusumillipraveen — APPROVED** [review]

**Review by mario-paniccia — APPROVED** [review]
👍

**Pull request merged**